### PR TITLE
Add RAL as GSOC organisation

### PR DESCRIPTION
**Pull request by StewMH**
Adding RAL as a new GSOC organisation. It wasn't clear to me if I needed to do this in the same step as making proposals - hopefully this is OK.

Cheers,
Stewart

### CODE REVIEW

**Comment by StewMH** [comment]
@graeme-a-stewart not sure why the CI is failing, looks unrelated to my changes

**Comment by graeme-a-stewart** [comment]
It is - Rucio has a broken certificate (which Chris and Mario assure me is being fixed).

**Comment by StewMH** [comment]
Hi Graeme,
Updated to the standard STFC logo, which is a bit less busy.

File: _activities/gsoc.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Instructions for participating projects and mentors can be found [here](/gsoc/gu
 | ![Metropolitan Institute of Technology](/images/logo_itm.png){:width="100px"} | ITM is an accredited public University, founded in 1944, in Medellin Colombia. [List of proposals](/gsoc/itm.html)|
 | ![Universitat Jaume I](/images/uji_logo.jpg){:width="100px"} |The Universitat Jaume I is a public university in Castelló de la Plana, Spain. [List of proposals](/gsoc/uji.html)|
 | ![Indiana University](/images/iu_logo.png){:width="100px"} | [Indiana University] (https://www.indiana.edu/), Bloomington. [List of proposals](/gsoc/iu.html)|
+| ![RAL](/images/logo_RAL.png){:width="100px"} | STFC Rutherford Appleton Laboratory in Oxfordshire, United Kingdom is home to national facilities including the Diamond synchroton and ISIS neutron spallation source. The Particle Physics Department contributes to experiments including ATLAS, CMS and T2K. [List of proposals](/gsoc/ral.html)|
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: You uploaded logo_RAL.jpg but ask for logo_RAL.png!

**Comment by graeme-a-stewart** [comment]
The story of this images directory traumatises me - it contains huge files that should never have been accepted and are used scaled down. Having been hurt before, I am extra sensitive to this now...

File: _activities/gsoc.md (hunk 1 above)
Comment (StewMH) [reply _activities/gsoc.md]: Sorry, fixed. Will also make the logo smaller.

Comment (StewMH) [reply _activities/gsoc.md]: convert -strip -interlace Plane -gaussian-blur 0.05 -quality 70% images/logo_RAL.jpg images/logo_RAL.jpg

has given me a 32k image :-)

Comment (graeme-a-stewart) [reply _activities/gsoc.md]: I would just use Preview and resize, but whatever works!

**Pull request merged**